packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -107,6 +107,45 @@ class CozyStackClient {
     }
   }
 
+  /**
+   * Retrieves a new app token by refreshing the currently used token.
+   * @throws {Error} The client should already have an access token to use this function
+   * @throws {Error} The client couldn't fetch a new token
+   * @returns {Promise} A promise that resolves with a new AccessToken object
+   */
+  async refreshToken() {
+    if (!this.token) throw new Error('No token to refresh')
```

Comment:
Would the error message be obvious for everybody? Maybe something like `Cannot refresh token when client does not have one at first`?